cmd/pingpong/runCmd.go

```diff
@@ -304,6 +306,16 @@ var runCmd = &cobra.Command{
 			cfg.AppLocalKeys = appProgLocalKeys
 		}
 
+		if numBox != 0 && numApp == 0 {
+			reportErrorf("If number of boxes is nonzero than number of apps must also be nonzero")
+		}
+
+		if numBox <= 8 {
```

Comment:
@algoidurovic Perhaps as an inline comment, can you help me understand why the limit is 8?

Reply:
The box size is hardcoded at 1024 so each box ref contributes exactly as much IO Budget as is needed, and the maximum number of box references is 8. I just thought that would be an easier model to work with but I admit it's somewhat rigid. If we decide to allow the user to set the box size then more logic needs to be added to compute the necessary number of box refs for the given number of boxes and their sizes.

Reply:
@algoidurovic Thanks for the context - I'm resolving though I optionally suggest adding an inline comment with similar content.  I think it'll improve comprehension for future maintainers.